### ISSUE TITLE
Helium font color at theme

### ISSUE THREAD

**Issue by nkorop** [gantry/gantry5#3276]
Hi!

We have an issue since Joomla 4 and continuing at 5... On configuration of theme at sections the font color is white and background light grey so we cannot read the title. We are forced to select all with CTrl+A to see them...

Any suggestion?

![Screenshot Helium](https://github.com/user-attachments/assets/2ad56c0b-3564-426b-93a9-20bcc852e8a9)

**Comment by N8Solutions** [comment]
@nkorop & @universewrld,
I use Joomla with Gantry 5 and I use a lot of their templates and I have never ran into this problem with any of them at all. I just did a fresh install of Joomla v5.2 with Gantry v5.5.19 and still don't see this issue.

Let me know what CMS you guys are using so I can see if I can reproduce it. You also need to provide steps to reproduce it, otherwise nothing can be done to fix it. The guys at Gantry need more to go on than just a photo with a complaint attached.

**Comment by N8Solutions** [comment]
> I think most of the admins use `Dark Theme` while working in `Control Panel`, I even saw research that in social networks more than 60% of users use Dark Theme all the time, so it is a global problem for @gantry.

@universewrld This is an issue caused by Joomla, not Gantry. They first introduced Dark Mode in the Joomla 4 admin template with no way to disable it. So if your PC is using Light or Dark mode, the admin area would follow suite because it uses the BS5 mechanism for auto dark mode. A lot of people complained that there was no way to shut it off when they first introduced it because it created a lot of problems because a lot of developers create their own interface WITHOUT using BS5. Hence the issue you ended up dealing with.

There is a way to add add an extension to Joomla 4 that will allow you to disable dark mode in the admin area as Joomla still hasn't fixed this in Joomla 4. However, in Joomla 5, the user menu has an option now where you can enable/disable Light/Dark Mode.

So if you are going to throw blame, make sure it is directed appropriately at Joomla for this as this is not a problem on Gantry's side. This is an issue that arose because the folks at Joomla decided not to provide an option to disable dark mode in Joomla 4 without thinking of the the chaos it would cause.

As I mentioned before, your issue is not an issue at all. It can become a feature request to improve Gantry 5. If you are able, and have the time, contribute to the project and provide the changes necessary to make it BS5 Dark Mode compatible.

[https://github.com/joomla/joomla-cms/issues/42134](https://github.com/joomla/joomla-cms/issues/42134)

[https://kevinsguides.com/blog/i-do-not-like-how-joomla-5-handles-dark-mode/](https://kevinsguides.com/blog/i-do-not-like-how-joomla-5-handles-dark-mode/)

To add to this, you can read what Nicholas K. Dionysopoulos from Akeeba Backup wrote here:
[https://github.com/nikosdion/DarkMagic](https://github.com/nikosdion/DarkMagic)

To sum things up, and to reiterate my point, the way the Joomla team implemented Dark Mode is bad. Gantry is not at fault here.

**Comment by N8Solutions** [comment]
> No, I don't want to disable the Dark Theme, I want the Dark Theme in Gantry 5. I didn't enable the Dark Theme on my system and browser to disable it.

@universewrld If you are going to do website design then you will need to disable Dark Mode in you browser while you do development work. The other option is for you to contribute to the Gantry 5 project by submitting a pull request with your proposal on how to add Dark Mode to Gantry 5.

**Comment by nkorop** [comment]
Guys,
The issue is not we use dark theme or not... the job to be done and all other things are decoration...

@N8Solutions thank you for the solution! Really healed my headache!

**Comment by N8Solutions** [comment]
@nkorop You're welcome. Glad I could help. I agree that it is a tool for us to be able to style a site correctly. While I would appreciate the Gantry 5 admin area being compatible with Dark Mode, what I'd like to see more is the ability to style a website for Dark Mode directly from the backend in Gantry 5. This is something I brought up to @mahagr back in 2020 with #2595.

**Comment by N8Solutions** [comment]
@universewrld you are incorrect. If it was only a Gantry 5 issue then there wouldn't be any discussion about how Joomla implemented Dark Mode as I pointed out in one of my previous responses.

**Comment by N8Solutions** [comment]
Again, your assumption is incorrect. If you read the GitHub information for the Dark Magic joomla integration I provide by Niko from Akeeba Backup you would understand why the way Joomla implemented it is an issue.
Also, there is no toggle for Light/Dark mode in Joomla 4 admin area. That is only available in Joomla 5.

**Comment by N8Solutions** [comment]
@universewrld Merely trying to get you to see the light of day about how your assumptions are wrong, and so others don't take your word on it when you are clearly wrong.

**Comment by N8Solutions** [comment]
> what was i wrong about? Gantry 5 doesn't have `Dark Mode`, isn't that true?

True, Gantry 5 doesn't have Dark Mode but you are wrong in that it is a "problem" with Gantry 5, it is not a bug. What you are asking for is a feature request.

I'm only trying to help you but you seem intent on throwing blame where it doesn't belong, and it also seems that you have an axe to grind with folks at Gantry 5 and Rocket Theme. I'm simply correcting your statements so others with less experience using Gantry 5 know what is going on.

**Comment by N8Solutions** [comment]
@universewrld the lack of a feature does not make it a :bug: :cockroach: :beetle: :fly: :roll_eyes:
You need not confuse the two.

**Comment by N8Solutions** [comment]
@universewrld Here's your official answer on Dark Mode from Rocket Theme.
[https://rockettheme.com/forum/gantry5-for-joomla/291041-dark-theme-for-gantry-5#1458866](https://rockettheme.com/forum/gantry5-for-joomla/291041-dark-theme-for-gantry-5#1458866).

As I said previously, it is not a bug or an error.
Just thought you'd appreciate hearing (reading) it from the horses mouth.

> It's not an error, it's just not compatible. Joomla requires 3rd party devs to write their own CSS if they want to be dark mode compatible.
> 
>Gantry is an open source project. You're welcome to submit the CSS as pull request here:[ github.com/gantry/gantry5/pulls](https://github.com/gantry/gantry5/pulls)
